NEW-STANDARD.md

```diff
@@ -0,0 +1,11 @@
+When adding a new WHATWG standard, these are the things that need to be done:
+
+1. Create a logo.
+1. Identify a shortname that is used for the spec.whatwg.org subdomain and repository (for tooling it is important that these are named identically).
+   1. Ensure the subdomain works.
+   1. Ensure there is repository.
+1. Create a Twitter account.
+1. Add it to https://github.com/whatwg/sg/blob/main/db.json and ensure the relevant details are filled in. (In case there's a new Workstream more updates will be needed.)
+1. Run https://github.com/whatwg/spec-factory.
```

Comment:
Why is this necessary? Can it be used to create a new empty repo in a previous step?

Reply:
It will (should?) fill in some of the boilerplate files.